examples/Text-To-English-Audio/Text To English Audio.md

```diff
@@ -0,0 +1,25 @@
+## Text To English Audio
+
+This Ballerina program takes a text file of any language from a given URL, transcribes the text into audio using OpenAI's Whisper model, translates the generated audio into English, and then converts the translated English text back into audio. Finally, it saves the translated audio as an .mp3 file on the local system. This use case is useful for creating podcasts or audio content in English from text available in another language.
```

Comment:
Same as above. How about "International news translator"? lets update the description and the variable names in the examples accordingly